app/components/isp-jam-auth/template.hbs

```diff
@@ -13,7 +13,7 @@
         </div>
       </div>
       <div class="form-group">
-        <button disabled={{if (or authenticating (not (and password username))) true false}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
+        <button disabled={{disabled}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
```

Comment:
`isp-jam-auth` extends `jam-auth`, where all of those properties are defined: https://github.com/CenterForOpenScience/exp-addons/blob/develop/exp-models/addon/components/jam-auth/component.js